libefiwrapper/storage.c

```diff
@@ -65,8 +65,8 @@ static EFI_GUID dp_guid = DEVICE_PATH_PROTOCOL;
 #define ABL_BDEVLIST "ABL.bootdevices"
 
 static boot_dev_t boot_dev = {
-	.type = STORAGE_EMMC,
```

Comment:
In this change, EMMC will not be supported any more?
Can we support both EMMC and NVME?